### PR TITLE
feat: export to sketch and pdf

### PR DESCRIPTION
**Pull request by TheReincarnator**
This is a pull request to add exporting of the current page to sketch or PDF.
This PR is not complete yet, as it depends on another module not publish to npm yet.

### CODE REVIEW

File: src/electron/menu.ts [hunk 1]
```diff
@@ -41,7 +69,7 @@ export function createMenu(): void {
 									designkitPath,
 									PathUtils.join(filePaths[0], 'designkit')
 								);
-								store.openStyleguide(`${filePaths[0]}/designkit`);
+								store.openStyleguide(`${filePaths[0]} / designkit`);
```
Comment (Jumace) [inline src/electron/menu.ts]: Are the added spaces here right? As it's a path I'm pretty sure we don't want them here

File: src/export/sketch-exporter.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+		page.addLayer(pageGroup);
+
+		pageGroup.addLayer(rootLayer);
+
+		FileExtraUtils.writeFileSync(path, JSON.stringify(page.toJSON(), null, '\t'));
+	}
+
+	// tslint:disable-next-line:no-any
+	private static isNotEmpty(value: any): boolean {
+		return value !== undefined && value !== null && value === '';
```
Comment (Jumace) [inline src/export/sketch-exporter.ts]: You could simplify this code a bit:
`return Boolean(value)`
This works for `undefined`, `''` and `null`

Comment (TheReincarnator) [reply src/export/sketch-exporter.ts]: Unfortunatelly, Boolean(value) issues `false` for `0`, or is there any good standard "is-empty" check in JavaScript, including strings?

**Comment by TheReincarnator** [comment]
@Jumace No worries, I now split the commit into three, and yes, the spaces in the path were an error, thanks for spotting!

File: src/export/sketch-exporter.ts (hunk 2 above)
Comment (Jumace) [reply src/export/sketch-exporter.ts]: Oh okay in that case your code is probably the best solution. Afaik there is no function like `isEmpty`.

**Comment by marionebl** [comment]
🎉

**Pull request merged**